docs/monoDrive_home/Simulator-Commands.md

```diff
@@ -0,0 +1,365 @@
+				"AutoExposureMaxBrightness": (float) - auto-exposure maximum adaptation,
+				"AutoExposureBias": (float) - logarithmic adjustment of the exposure
+			}
+		},
+		...  - additional profiles
+	]
+}
+```
+
+**response**: (string)
+
+`DynamicWeather is not available` if the weather actor is missing from the map
+`Weather configuration failed` if the weather profiles are invalid or mal-formed
+`Weather profile is invalid or missing` if the profile ID is not found
+`Weather configured to {profile_id}` if the weather was properly configured
+
+
+## Sensor Configuration
+Configures a sensor or set of sensors to use with the ego vehicle in the simulation run. The command can take a [single sensor configuration](Common.md), or an array of sensor configurations.
+
+**command ID**:  "REPLAY_ConfigureSensorsCommand_ID"
+
+**command data**:
+
+[sensor_config](Common.md)
+
+```json
+[
+	Sensor_config, ...
+]
+```
+
+**response**: (string)
+
+`complete` if the sensor was properly configured
+
+
+## Sensor Reconfiguration
+Reconfigures a previously configured sensor or set of sensors to use with the ego vehicle in the simulation run. The command takes a [single sensor configuration](Common.md).
+
+**command ID**:  "REPLAY_ReConfigureSensorCommand_ID"
+
+**command data**:
+
+[sensor_config](Common.md)
+
+**response**: (string)
+
+`complete` if the sensor was properly reconfigured
+
+
+## Trajectory Configuration
+Configures the simulation [trajectory](../../scenario_editor/trajectory_files). The trajectory file determines the initial state of the simulation for closed loop mode, and a series of frames for replay/replay step mode. The frames provide the location/state of all actors in the simulation as a given point.
+
+**command ID**:  "REPLAY_ConfigureTrajectoryCommand_ID"
+
+**command data**:
+
+[trajectory](../../scenario_editor/trajectory_files)
+
+**response**: (string)
+
+`complete` if the trajectory was properly applied
+
+
+## Step the Simulation
+Steps the simulation (either forward or backward) by the specified number of frames using the previously configured trajectory.
+
+**command ID**:  "REPLAY_StepSimulationCommand_ID"
+
+**command data**:
+```json
+{
+	amount: (integer) - the number of steps to advance the simulation by
+}
+```
+**response**: (string)
+
+`complete` if the simulation was advanced properly
+
+## State Step Simulation
+Sets the simulation frame to the provided frame.
+
+**command ID**:  "REPLAY_StateStepSimulationCommand_ID"
+
+**command data**:
+
+[A state_frame object](../../scenario_editor/trajectory_files/#replaying-trajectory-files)
+
+**response**: (string)
+
+`complete` if the simulation was advanced properly
+
+
+## Get Start Points
+Retrieves the set of starting locations for the current simulation map.
+
+**command ID**:  "GetStartPoints"
+
+**command data**:
+```json
+{
+}
+```
+
+**response**: (json object)
+``` json
+{
+	"type": [ "PIE" | "startPlayer", ... ],
```

Comment:
What does the `...` mean here? Are there more modes? Why don't we list them here?